presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -147,6 +147,17 @@ Accessors
     Returns the 2-dimensional cartesian minimum distance (based on spatial ref)
     between two geometries in projected units.
 
+.. function:: ST_GeometryN(Geometry, index) -> Geometry
+
+    If the geometry is a collection of geometries (e.g., GEOMETRYCOLLECTION or MULTI*),
```

Comment:
All the functions in this doc starts with `Returns ...`. We should be consistent, so please have the same structure for these functions too.